test/jdk/com/sun/java/swing/plaf/gtk/bug6492108.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Component;
+import java.awt.Container;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.image.BufferedImage;
+
+import javax.swing.JEditorPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+import javax.swing.JTextPane;
+import javax.swing.UIManager;
+import javax.swing.text.JTextComponent;
+
+/*
+ * @test
+ * @bug 6492108 8160755
+ * @key headful
+ * @requires (os.family == "linux")
+ * @summary Verifies that the background is painted the same for
+ *          JTextArea, JTextPane, and JEditorPane.
+ * @library /javax/swing/regtesthelpers
+ * @build SwingTestHelper
+ * @run main/othervm bug6492108
+ */
+
+public class bug6492108 extends SwingTestHelper {
+
+    private JPanel panel;
+
+    public static void main(String[] args) throws Throwable {
+        try {
+            UIManager.setLookAndFeel(
+                "com.sun.java.swing.plaf.gtk.GTKLookAndFeel");
+        } catch (Exception e) {
+            System.out.println("GTK LAF is not supported on this system; test passes");
+            return;
+        }
+        new bug6492108().run(args);
+    }
+
+    private static void addTextComps(Container parent,
+                                     Class<? extends JTextComponent> type)
+        throws Throwable
+    {
```

Comment:
I think formatting here looks a little odd, could probably move the bracket onto the same line as `throws Throwable` and shift it over to the right to match `Class<? extends JTextComponent> type)`

Reply:
Updated.

Reply:
Previous formatting was less confusing and aligned with Java Coding Style Guidelines.

The parameters to the method are aligned to the opening parenthesis; the `throws` clause is not part of the parameters and it's placed on another line according to rules of wrapping lines (it should've used double-indentation of 8 spaces rather than 4 spaces to indicate a continuation line).

Reply:
Reverted back to previous formatting with double-indentation. Hopefully this should be ok now.

Reply:
This formatting looks good to me.